notify-delivery/src/notify_delivery/resources/gc_notify.py

```diff
@@ -0,0 +1,76 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions andcurrent_app.
+# limitations under the License.
+"""Worker resource to handle incoming queue pushes from gcp."""
+import sys
+from http import HTTPStatus
+
+from flask import Blueprint, request
+from notify_api.models import Notification, NotificationHistory, NotificationSendResponses
+from notify_api.services.gcp_queue import queue
+from notify_api.utils.logging import logger
+
+from notify_delivery.services.providers import _all_providers  # noqa: E402
+
+bp = Blueprint("gcnotify", __name__)
+
+
+@bp.route("/", methods=("POST",))
```

Comment:
do we need to verify that message was sent by gcp via @ensure_authorized_queue_user

Reply:
The delivery service is authenticated internally through a service account, and I don't think it's necessary to authorize through a JWT token.

![image](https://github.com/bcgov/bcregistry-sre/assets/44277752/c9792ede-9592-4118-8153-9e6e22a67463)